src/analyses/mutexEventsAnalysis.ml

```diff
@@ -82,6 +82,11 @@ struct
             ctx.split () [Events.Unlock (remove_fn e)]
           ) (eval_exp_addr (Analyses.ask_of_ctx ctx) arg);
         raise Analyses.Deadcode
+      | "LAP_Se_SignalSemaphore", [Lval arg; _] ->
+        List.iter (fun e ->
+            ctx.split () [Events.Unlock (remove_fn e)]
+          ) (eval_exp_addr (Analyses.ask_of_ctx ctx) (Cil.mkAddrOf arg));
+        raise Analyses.Deadcode
```

Comment:
Should we maybe make this conditional on (the/an) ARINC analysis being activated?
Just to be safe if we ever encounter a name clash?

Reply:
The name clash already occurs at the level of `LibraryFunctions` where we currently don't have ARINC (and Linux kernel) functions categorized properly for separate activation, where this should really be done. But I added extra checks here nevertheless.